src/database/db.js

```diff
@@ -0,0 +1,38 @@
+import { initDB } from "./connect";
+import { getTranslitText } from "./utils/utils";
+
+export function getBaniList(language) {
+  return new Promise((resolve, reject) => {
```

Comment:
It is preferred to use await/async function instead of manually creating Promise as it can simplifies the code and makes it easier to understand.

Reply:
I tried await/async but this library is 3-4 years old. which only allow us to use promises.